circleguard/widgets.py

```diff
@@ -1,14 +1,15 @@
 # pylint: disable=no-name-in-module
+from functools import partial
 from PyQt5.QtWidgets import (QWidget, QGridLayout, QLabel, QLineEdit,
                              QSpacerItem, QSizePolicy, QSlider, QSpinBox,
                              QDoubleSpinBox, QFileDialog, QPushButton)
 from PyQt5.QtGui import QRegExpValidator
-from PyQt5.QtCore import QRegExp, Qt, QSettings, QDir, QCoreApplication
+from PyQt5.QtCore import QRegExp, Qt, QDir, QCoreApplication
 # pylint: enable=no-name-in-module
-from functools import partial
 from settings import THRESHOLD
+# pylint: disable=no-name-in-module
 
-spacer = QSpacerItem(100, 0, QSizePolicy.Maximum, QSizePolicy.Minimum)
+SPACER = QSpacerItem(100, 0, QSizePolicy.Maximum, QSizePolicy.Minimum)
```

Comment:
this one is actually a good use of all caps, we want to emphasize the fact that it's unchanging and we're using it as is with no modifications.